test/ranking/ranking.test.ts

```diff
@@ -0,0 +1,36 @@
+import {Channel} from 'vega-lite/src/channel';
+import {Type} from 'vega-lite/src/type';
+
+import {schema} from '../fixture';
+import {rank} from '../../src/ranking/ranking';
+
+import {SpecQueryModelGroup} from '../../src/modelgroup';
+
+import {assert} from 'chai';
+
+
+describe('ranking', () => {
+  describe('rank', () => {
+    describe('group.items', () => {
```

Comment:
rank is the method you're testing.  

So `group.items` shouldn't be a describe instead put all of these inside `it('should return a empty group if input group is empty')`   (No need for sub-`it` below.)


(Basically group is empty is it has no items.)